include/swift/Reflection/ReflectionContext.h

```diff
@@ -31,6 +35,31 @@
 #include <vector>
 #include <unordered_map>
 
+#ifdef __APPLE__
```

Comment:
This code seems MachO specific.  Would you mind using:

    #if defined(__APPLE__) && defined(__MACH__)